src/components/Card/Card.stories.tsx

```diff
@@ -2,8 +2,7 @@ import * as React from "react";
 import { action } from "@storybook/addon-actions";
```

Comment:
This is a general comment about the story file.  Please convert to `.mdx` format and present the props similar to other `.mdx` stories.